tidb-lightning: update wording about parallel import (#20431)
pingcap/docs: tidb-lightning/tidb-lightning-physical-import-mode.md

- Do not use the physical import mode to directly import data to TiDB clusters in production. It has severe performance implications. If you need to do so, refer to [Pause scheduling on the table level](/tidb-lightning/tidb-lightning-physical-import-mode-usage.md#scope-of-pausing-scheduling-during-import).
- If your TiDB cluster has a latency-sensitive application and a low concurrency, it is strongly recommended that you **do not** use the physical import mode to import data into the cluster. This mode might have significant impact on the online application.

> - If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, enable [parallel import](/tidb-lightning/tidb-lightning-distributed-import.md).

Reviewer: ```suggestion
- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, better enable [parallel import](/tidb-lightning/tidb-lightning-distributed-import.md).
```
We actually support running multiple TiDB Lightning instances without `parallel import` enabled, but from the original context, it seems a suggestion

> - If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, enable [parallel import](/tidb-lightning/tidb-lightning-distributed-import.md).

Reviewer 2: ```suggestion
- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, you can enable [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) to coordinate the import. If each instance imports different tables, the Parallel Import option is not required. However, if multiple instances import data into the same table, enabling Parallel Import is necessary to prevent conflicts and ensure data integrity.
```
cc @D3Hunter

> - If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, you can enable [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) to coordinate the import. If each instance imports different tables, the Parallel Import option is not required. However, if multiple instances import data into the same table, enabling Parallel Import is necessary to prevent conflicts and ensure data integrity.

Author: ```suggestion
- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, you can enable [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) to coordinate the import. If each instance imports data into different tables, the Parallel Import option is not required. However, if multiple instances import data into the same table, you need to enable Parallel Import to prevent conflicts and ensure data integrity.
```

> - If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, you can enable [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) to coordinate the import. If each instance imports data into different tables, the Parallel Import option is not required. However, if multiple instances import data into the same table, you need to enable Parallel Import to prevent conflicts and ensure data integrity.

Reviewer 3: ```suggestion
- If you want to run multiple TiDB Lightning instances simultaneously to import data into the same TiDB cluster, you can enable [Parallel Import](/tidb-lightning/tidb-lightning-distributed-import.md) to coordinate the import. If each instance imports data into **different tables**, the Parallel Import option is not required. However, if multiple instances import data into **the same table**, you need to enable Parallel Import to prevent conflicts and ensure data integrity.
```
